### ISSUE TITLE
Bug: font-family-fallbacks treats var() function names as case-sensitive

### ISSUE THREAD

**Issue by KumJungMin** [eslint/css#534]
### Environment

ESLint version: 10.8.0
@eslint/css version: 1.4.0
Node version: 22.18.0
npm version: 10.9.3
Operating System: Darwin 24.6.0

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```js
import { defineConfig } from "eslint/config";
import css from "@eslint/css";

export default defineConfig([
    {
        files: ["**/*.css"],
        language: "css/css",
        plugins: { css },
        rules: {
            "css/font-family-fallbacks": "error",
        },
    },
]);
```

</details>

```css
:root {
    --font-family: Arial, sans-serif;
}

/* Correct: No error is reported */
.lowercase {
    font-family: var(--font-family);
}

/* Bug 1: Reports a useFallbackFonts error despite referencing the same value */
.uppercase {
    font-family: VAR(--font-family);
}

/* Bug 2: Reports the same error for a mixed-case function name */
.mixed-case {
    font-family: vAr(--font-family);
}
```

### What did you expect to happen?

CSS function names are ASCII case-insensitive, so uppercase and mixed-case forms such as `VAR()`, `Var()`, and `vAr()` should behave the same as lowercase `var()`.

None of the following declarations should report an error because `--font-family` contains a generic font fallback:

```css
.lowercase {
    font-family: var(--font-family);
}

.uppercase {
    font-family: VAR(--font-family);
}

.mixed-case {
    font-family: vAr(--font-family);
}
```

### What actually happened?

`font-family-fallbacks` recognizes only lowercase `var()` as a CSS variable function. As a result, the declarations using `VAR()` and `vAr()` incorrectly report a `useFallbackFonts` error:

```text
/home/project/test.css
  12:18  error  Use fallback fonts and a generic font last  css/font-family-fallbacks
  17:18  error  Use fallback fonts and a generic font last  css/font-family-fallbacks
```

### Link to Minimal Reproducible Example

[https://stackblitz.com/edit/vitejs-vite-cow7garn?file=test.css](https://stackblitz.com/edit/vitejs-vite-cow7garn?file=test.css)

### Participation

* [x] I am willing to submit a pull request for this issue.

### AI acknowledgment

* [ ] I did not use AI to generate this issue report.
* [x] (If the above is not checked) I have reviewed the AI-generated content before submitting.

### Additional comments
> Disclosure: I'm a participant in the [OSSCA open source contribution program](https://github.com/eslint-ossca).

This is the same type of issue that was fixed for `no-invalid-properties` in [PR #232](https://github.com/eslint/css/pull/232).

PR #232 fixed cases such as:

```css
a {
    color: VAR(--color);
}
```

The same case-sensitive comparisons remain in `font-family-fallbacks`.

The rule currently contains direct comparisons such as:

```js
node.type === "Function" && node.name === "var"
```

It also contains string-based checks such as:

```js
value.startsWith("var")
```

These checks do not recognize uppercase or mixed-case function names.

A possible solution is to make `isVarFunction()` case-insensitive:

```js
function isVarFunction(node) {
    return node.type === "Function" && node.name.toLowerCase() === "var";
}
```

The remaining direct and string-based checks could then be replaced with AST-based calls to `isVarFunction()`.

This change should apply only to the `var` function name. Custom property names such as `--font` and `--FONT` must remain case-sensitive.

The CSS Values and Units specification states that function names are ASCII case-insensitive:

[[CSS Values and Units Level 4: Functional Notations](https://www.w3.org/TR/css-values-4/#functional-notations)](https://www.w3.org/TR/css-values-4/#functional-notations)

**Comment by DMartens** [comment]
The case-insensitivity for the remaining rules (including `font-family-fallbacks`) is currently being done in #530.

**Comment by KumJungMin** [comment]
> The case-insensitivity for the remaining rules (including `font-family-fallbacks`) is currently being done in [#530](https://github.com/eslint/css/pull/530).

---

I missed that this was already being addressed in #530. 
I’ll be sure to double-check open PRs before submitting future issues. Thanks for pointing it out! :)
